tests/page_test.go

```diff
@@ -182,6 +179,111 @@ func TestPageGotoWaitUntilDOMContentLoaded(t *testing.T) {
 	assert.EqualValues(t, "DOMContentLoaded", actual[0], `expected "DOMContentLoaded" event to have fired`)
 }
 
+func TestPageInnerHTML(t *testing.T) {
+	t.Parallel()
+
+	t.Run("ok", func(t *testing.T) {
+		t.Parallel()
+
+		p := newTestBrowser(t).NewPage(nil)
+		p.SetContent(sampleHTML, nil)
+		assert.Equal(t, `<b>Test</b><ol><li><i>One</i></li></ol>`, p.InnerHTML("div", nil))
+	})
+
+	t.Run("err_empty_selector", func(t *testing.T) {
+		t.Parallel()
+
+		defer func() {
+			assertPanicErrorContains(t, recover(), "The provided selector is empty")
+		}()
+
+		p := newTestBrowser(t).NewPage(nil)
+		p.InnerHTML("", nil)
+		t.Error("did not panic")
+	})
+
+	t.Run("err_wrong_selector", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		p := tb.NewPage(nil)
+		p.SetContent(sampleHTML, nil)
+		assert.Equal(t, "", p.InnerHTML("p", tb.rt.ToValue(jsFrameBaseOpts{
+			Timeout: "100",
+		})))
+	})
+}
+
+func TestPageInnerText(t *testing.T) {
+	t.Parallel()
+
+	t.Run("ok", func(t *testing.T) {
+		t.Parallel()
+
+		p := newTestBrowser(t).NewPage(nil)
+		p.SetContent(sampleHTML, nil)
+		assert.Equal(t, "Test\nOne", p.InnerText("div", nil))
+	})
+
+	t.Run("err_empty_selector", func(t *testing.T) {
+		t.Parallel()
+
+		defer func() {
+			assertPanicErrorContains(t, recover(), "The provided selector is empty")
+		}()
+
+		p := newTestBrowser(t).NewPage(nil)
+		p.InnerText("", nil)
+		t.Error("did not panic")
```

Comment:
Maybe?
```golang
const emptySelector = ""
p.InnerText(emptySelector, nil)
```

Reply:
Isn't this already clear from the test name? Adding a single use constant seems pointless.